node-notes-cli/app.js

```diff
@@ -0,0 +1,31 @@
+const file = './data.json';
+const data = require(file);
+const fs = require('fs');
+const action = process.argv[2];
+const input = process.argv[3];
+const update = process.argv[4];
+
+fs.writeFile(file, JSON.stringify(data, null, 2), err => {
```

Comment:
You don't need this outer `writeFile`, you only need to update the file after there has been changes to the data object.

Reply:
fixed!